city_scrapers/spiders/tulok_bocc.py

```
@@ -0,0 +1,150 @@
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class TulokBoccSpider(CityScrapersSpider):
+    name = "tulok_bocc"
+    agency = "Tulsa Board of County Commissioners"
+    timezone = "America/Chicago"
+    api_base_url = "https://tulsacook.api.civicclerk.com"
+    portal_base_url = "https://tulsacook.portal.civicclerk.com"
+    info_page_url = "https://www2.tulsacounty.org/Legacy/agendasdetail_civic.aspx?entity=BOCC%20-%20Board%20of%20County%20Commissioners"  # noqa
```

Ruff (0.14.8)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)